### ISSUE TITLE
MobileNet v1 example

### ISSUE THREAD

**Issue by texasmichelle** [tensorflow/swift-models#239]
Add an example that uses the MobileNet v1 model and the ImageNette dataset.

Blocked by #238, #237.

**Comment by BradLarson** [comment]
Completed via PR #240, with training being verified in PR #241. For posterity, these are the results I see (against the test set) in the first ten epochs of training with Imagenette:

> [Epoch 1] Accuracy: 215/500 (0.43) Loss: 1.7700883
> [Epoch 2] Accuracy: 282/500 (0.564) Loss: 1.4064155
> [Epoch 3] Accuracy: 250/500 (0.5) Loss: 1.664016
> [Epoch 4] Accuracy: 304/500 (0.608) Loss: 1.2813418
> [Epoch 5] Accuracy: 352/500 (0.704) Loss: 0.9346338
> [Epoch 6] Accuracy: 373/500 (0.746) Loss: 0.80547553
> [Epoch 7] Accuracy: 367/500 (0.734) Loss: 0.8665428
> [Epoch 8] Accuracy: 386/500 (0.772) Loss: 0.71155274
> [Epoch 9] Accuracy: 348/500 (0.696) Loss: 1.0265149
> [Epoch 10] Accuracy: 405/500 (0.81) Loss: 0.6340269
